app.js

```diff
@@ -1,17 +1,67 @@
 var express = require('express');
 var databaseConfig = require('./config/databaseConfig');
+var flash = require('connect-flash');
+var passport = require('passport'),
+  LocalStrategy = require('passport-local').Strategy;
+var session = require('express-session');
 var app = express();
-
 var mongoose = require('mongoose');
 mongoose.connect(databaseConfig.getMongoURI());
 
 app.use(express.static('public'));
 app.use(express.static('node_modules'));
+app.use(flash());
+app.use(session({
+  secret: 'secretkey1'
```

Comment:
secret key used to sign session ids (i.e. we can verify that the session was produced by our server)